functions/service-account.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What is the purpose of this file?

Reply:
provide Service Account credential in the LinkedIn function https://github.com/newjersey/career-network/pull/127/files#diff-1e53ef2f51bd446c876676ba83d7c888